content/en/real_user_monitoring/guide/sampling-browser-plans.md

```diff
@@ -44,7 +44,7 @@ To track 100% of your sessions as Browser RUM:
 ```
 datadogRum.init({
     ....
-    sampleRate: 100,
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Introduce a `before v4.30.0` section to keep track of the history